nautobot/circuits/signals.py

```diff
@@ -1,8 +1,35 @@
+from cacheops import invalidate_obj
+from django.contrib.contenttypes.models import ContentType
+
```

Comment:
I know this is is a trivial style nit, but can you please ditch this extra blank line here?